module/helper/renderer/src/webgl/renderer.rs

```diff
@@ -70,6 +81,10 @@ mod private
     pub transparent_revealage_texture : Option< gl::web_sys::WebGlTexture >,
     #[ allow( dead_code ) ]
     pub depth_renderbuffer : Option< gl::web_sys::WebGlRenderbuffer >,
+    /// The framebuffer used for rendering skybox
+    pub skybox_framebuffer : Option< gl::web_sys::WebGlFramebuffer >,
+    /// Texture with equirectangular map
+    pub skybox_texture : Option< gl::web_sys::WebGlTexture >,
```

Comment:
Why do we need a separate framebuffer for the skybox?
And why the texture is part of the framebuffer struct and not the renderer?